javascript/javascript1/week2/student.js

```diff
@@ -0,0 +1,31 @@
+const class07Students = [];
+function addStudentToClass(studentName) {
+  // statement values and outputs
+  if (class07Students.length >= 6) {
+    console.log("cannot add more student to class 07");
+  } else if (class07Students.includes(studentName)) {
+    console.log("Student " + studentName + " is already an existing student ");
+  } else if (class07Students === "Queen") {
+    class07Students.push(studentName);
+  } else if (studentName === "") {
```

Comment:
what about multiple spaces `"   "` you can use `trim()` to remove whitespaces so it will work if there is one or more spaces
```js
} else if (studentName.trim() === "") {
```